README.md

```diff
@@ -79,7 +79,8 @@ Contributing security advisories is as easy as it can get:
 
   * Make sure your file validates by running `php validator.php` from the root of this project.
     This script needs some dependencies to be installed via composer, so you need to
-    run `composer install` before.
+    run `composer install` before. The validator uses a lot of memory, so you might need to
+    use `-d memory_limit=...` (e.g. `php -d memory_limit=200M validator.php`).
```

Comment:
What about `memory_limit=-1`?

Reply:
Perhaps just
```diff
-   * Make sure your file validates by running `php validator.php` from the root of this project.
+   * Make sure your file validates by running `php -d memory_limit=-1 validator.php` from the root of this project.
```

in that case? (skip the qualifying statement about using a lot of memory)

Reply:
Works for me